src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1

```diff
@@ -747,3 +747,16 @@ function Verify-PSOperationStatusResponse
     Assert-Null $result.Error;
 }
 
+<# 
+.SYNOPSIS
+Sleeps but only during recording.
+#>
+ 
+function Start-TestSleep($milliseconds)
```

Comment:
Is there a reason not to use Wait-Seconds? https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L307